server/i18n/en.json

```
@@ -2630,15 +2630,15 @@
   },
   {
     "id": "api.post.disabled_all",
-    "translation": "@all has been disabled because the channel has more than {{.Users}} users."
+    "translation": "@all notifications are disabled in channels with more than {{.Users}} users to maintain server performance. Contact your system administrator for more information."
   },
   {
     "id": "api.post.disabled_channel",
-    "translation": "@channel has been disabled because the channel has more than {{.Users}} users."
+    "translation": "@channel notifications are disabled in channels with more than {{.Users}} users to maintain server performance. Contact your system administrator for more information."
```

I don't think we need to explain the performance issue to end users.
```suggestion
    "translation": "@channel notifications are disabled in channels with more than {{.Users}} users. Contact your system administrator for more information."
```